frontend/scss/molecules/_m-feature-block.scss

```diff
@@ -0,0 +1,273 @@
+// padding-top: 4.6875vw;
+// padding-top: 2.34375vw;
+// padding-top: 1.5625vw;
+
+
+.m-feature-block-heading {
+    margin-top: 68px;
+    border-top: 1px solid #e6e6e6;
```

Comment:
Swap out the colors in this file with a color variable. In the case of `#e6e6e6` here it would be `$color__rules--primary`